server/model/repo.go

```diff
@@ -99,3 +104,9 @@ type RepoPatch struct {
 	AllowPull                    *bool           `json:"allow_pr,omitempty"`
 	CancelPreviousPipelineEvents *[]WebhookEvent `json:"cancel_previous_pipeline_events"`
 }
+
+type RemoteID string
```

Comment:
with this we could in future make it an interface and do more granular validation per remote if we ever need it, else it's just make sure the logic is only implemented once -> IsSet()

Reply:
PS: the whole conversion from and to RemoteID from int64... we could also write a helper function if desired ...